### PR TITLE
API1200 support for Enclosure, NetworkSet, SP and Task

### PR DESCRIPTION
**Pull request by AnushaY1916**
### Description
API 1200 support for Enclosure, Network set, Server profile and Task

### Issues Resolved
NA

### Check List
- [x] New functionality includes testing.
  - [x] All tests pass for go 1.11 + gofmt checks.
- [ ] New functionality has been documented in the README if applicable.
  - [ ] New functionality has been thoroughly documented in the examples (please include helpful comments).
- [x] Changes are documented in the CHANGELOG.

### CODE REVIEW

File: examples/server_profile.go [hunk 1]
```diff
@@ -20,14 +20,14 @@ func main() {
...
-		800,
+		1200,
 		"*")
 
 	initialScopeUris := new([]utils.Nstring)
 	*initialScopeUris = append(*initialScopeUris, utils.NewNstring("/rest/scopes/74877630-9a22-4061-9db4-d12b6c4cfee0"))
 
 	server_profile_create_map := ov.ServerProfile{
-		Type:              "ServerProfileV9",
+		Type:              "ServerProfileV11",
```
Comment (soodpr) [inline examples/server_profile.go]: are we not adding new fields like consistencyState in the struct?

File: examples/enclosure.go [hunk 2]
```diff
@@ -20,7 +20,7 @@ func main() {
 		os.Getenv("ONEVIEW_OV_DOMAIN"),
 		os.Getenv("ONEVIEW_OV_ENDPOINT"),
 		false,
-		1000,
+		1200,
```
Comment (soodpr) [inline examples/enclosure.go]: same for enclosure, we have some new fields introduced for 1200 version

File: examples/server_profile.go (hunk 1 above)
Comment (AnushaY1916) [reply examples/server_profile.go]: No, we are just minimal support for 1200.

File: examples/enclosure.go (hunk 2 above)
Comment (AnushaY1916) [reply examples/enclosure.go]: No, we are just giving minimal support for 1200 with no changes to structs.

**Pull request merged**